meegkit/utils/trca.py

```diff
@@ -99,34 +99,36 @@ def itr(n, p, t):
     return itr
 
 
-def filterbank(eeg, fs, idx_fb):
+def bandpass(eeg, sfreq, Wp, Ws):
```

Comment:
Maybe, the only bandpass as the name of the function could be misleading, how about changing it with bandapass_filterbank to emphasize the use of the filterbank?

Reply:
Well this function **is** effectively a simple bandpass filter, and as such could be reused independently (i.e. outside the scope of the TRCA), which is why I wanted to use `bandpass`